### PR TITLE
add inclusive/exclusive mode in avg pool

### PR DESCRIPTION
**Pull request by heavengate**
Add an attribute "exclusive" to indicate whether to use exclusive mode with default value as true.
In exclusive mode, avg pool calc points without padding.

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -2044,6 +2044,7 @@ def pool2d(input,
            global_pooling=False,
            use_cudnn=True,
            ceil_mode=False,
+           exclusive=True,
```
Comment (qingqing01) [inline python/paddle/fluid/layers/nn.py]: For  forward compatibility:

```
name=None,
exclusive=True):
```

Comment (heavengate) [reply python/paddle/fluid/layers/nn.py]: Thanks for your suggestion and have fixed for pool2d and pool3d

**Review by qingqing01 — DISMISSED** [review]
LGTM.

File: paddle/fluid/operators/pool_op.cc [hunk 2]
```diff
@@ -180,6 +180,12 @@ void Pool2dOpMaker::Make() {
       "operator."
       "If global_pooling = true, paddings and ksize will be ignored.")
       .SetDefault({0, 0});
+  AddAttr<bool>(
```
Comment (panyx0718) [inline paddle/fluid/operators/pool_op.cc:183]: Also update "AddComment" so show the math when exclusive=False?

Comment (heavengate) [reply paddle/fluid/operators/pool_op.cc:183]: Done for add math in AddComment.

**Review by heavengate — COMMENTED** [review]
Thanks for reviewing and have done for add math formula in AddComment

**Review by panyx0718 — DISMISSED** [review]
LG API

**Review by qingqing01 — DISMISSED** [review]

**Review by qingqing01 — APPROVED** [review]

**Review by panyx0718 — APPROVED** [review]

**Pull request merged**